.github/workflows/publish.yml

```diff
@@ -14,6 +14,18 @@ jobs:
       with:
         node-version: 14.x
         cache: 'yarn'
+    - uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: "us-gov-west-1"
```

Comment:
We're doing the same thing in our `publish-storybook` workflow, so I copied that:

https://github.com/department-of-veterans-affairs/component-library/blob/e157e3c011afb64c8431cc7c7b266ab4c786ac03/.github/workflows/publish-storybook.yml#L34-L38